pcsx2/x86/iFPU.cpp

```diff
@@ -114,6 +114,7 @@ static const __aligned16 u32 s_pos[4] = { 0x7fffffff, 0xffffffff, 0xffffffff, 0x
 //------------------------------------------------------------------
 void recCFC1(void)
 {
+	EE::Profiler.EmitOp(eeOpcode::CFC1);
```

Comment:
put the emitop after the "check validity return" because the instruction could be a nop here. (Same for the call below).